log/slog.go

```diff
@@ -0,0 +1,188 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/cockroachdb/errors"
+	"github.com/cockroachdb/errors/withstack"
+	"golang.org/x/exp/slog"
+)
+
+type RelayLogger struct {
+	*slog.Logger
+}
+
+var relayLogger *RelayLogger
+
+func InitLogger(logLevel, format, output string) error {
+	// level
+	var slogLevel slog.Level
+	switch logLevel {
```

Comment:
I found https://pkg.go.dev/golang.org/x/exp/slog#Level.UnmarshalText. Does it match our purpose?

Reply:
@bluele Thank you! Yes, we can use this to improve the code.
```
var slogLevel slog.Level
if err := slogLevel.UnmarshalText([]byte(logLevel)); err != nil {
    return fmt.Errorf("failed to unmarshal level: %v", err)
}
```

Reply:
@dongrie Could you fix it using this function?

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/92/commits/46c50e288b43464120b99c7ba2e3299afd21410b
Fixed